client/src/App.js

```diff
@@ -2,12 +2,18 @@ import { Route, Routes } from "react-router-dom";
 
 import About from "./pages/About";
 import Home from "./pages/Home";
+import Login from "./pages/Login";
 
 const App = () => (
+	<div className="App">
+		{/* <Login /> */}
 	<Routes>
-		<Route path="/" element={<Home />} />
+		<Route path="/" element={<Login />} />
 		<Route path="/about/this/site" element={<About />} />
 	</Routes>
+	</div>
 );
```

Comment:
Could you please delete the unnecessary div and commented code
```js
<div className="App">
</div>

```

Reply:
I will definitely do that.